test_cosi.sh

```diff
@@ -10,9 +10,10 @@ tails=8
 main(){
```

Comment:
The 'bash-import' above should also depend on the cothority and not the cothority.v0